security-privacy-questionnaire.md

```diff
@@ -0,0 +1,105 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+Payment Link URIs are exposed by websites and received by payment clients, which are either integrated wallets within the user agent or vetted browser extensions installed by the user. These URIs include a scheme (e.g., upi, bitcoin, paypal) and parameters like payee address, amount, and currency. This information is necessary for the user agent to identify the payment method, initiate payment, and offer users payment options. The user agent mediates access to this information, giving users control over which payment clients are active and can access payment links through browser settings. The user agent will clearly indicate when a payment link is detected and which payment client can handle it, ensuring user awareness and consent. Payment clients are linked and authorized to be used through the user agent.
```

Comment:
`vetted` --> `opted in` (to use the language that is more common in the privacy community).

Please place the phrase "offer users payment options" before "initiate payment", so it's clear that the user has control before the payment happens.

Reply:
Thanks for helping reword.